src/main/java/org/gridsuite/study/server/StudyController.java

```diff
@@ -1386,7 +1426,8 @@ public ResponseEntity<Void> stopBuild(@Parameter(description = "Study uuid") @Pa
     public ResponseEntity<Void> changeModificationActiveState(@PathVariable("studyUuid") UUID studyUuid,
                                                                     @PathVariable("nodeUuid") UUID nodeUuid,
                                                                     @PathVariable("modificationUuid") UUID modificationUuid,
-                                                                    @Parameter(description = "active") @RequestParam("active") boolean active) {
+                                                                    @Parameter(description = "active") @RequestParam("active") boolean active,
+                                                                    @RequestHeader("userId") String userId) {
```

Comment:
userId is not used. Should be used in 
_studyService.changeModificationActiveState(studyUuid, nodeUuid, modificationUuid, active, **userId**);_

Reply:
fixed